packages/react/src/components/Card/CardContent.js

```diff
@@ -31,21 +25,38 @@ const mobileDescriptionStyle = css`
 
 export const Content = styled('div')`
```

Comment:
@Kikobeats I noticed there's a lot of `styled('element')...` - instead of `styled.element`. Should I unify it all as `styled.div` etc?